README.md

```diff
@@ -24,16 +24,30 @@ what configuration you need on the AWS side.
 
 ### General recommendations
 
-The number of `replicas` in the [default values file](helm/loki/values.yaml) are generally considered as safe.
+The number of `replicas` in the [default values file](https://github.com/giantswarm/loki-app/helm/loki/values.yaml) are generally considered as safe.
 If you reduce the number of `replicas` below the default recommended values, expect undefined behaviour and problems.
 
+#### Prepare config file
+
+1. Create app config file
+    Grab the included [sample config file](https://github.com/giantswarm/loki-app/sample_configs/values-gs.yaml) or [azure sample config file](https://github.com/giantswarm/loki-app/sample_configs/values-gs-azure.yaml) ,
+    read the comments for options and adjust to your needs. To check all available
+    options, please consult the [upstream `values.yaml` file](https://github.com/giantswarm/loki-app/helm/loki/values.yaml).
+
+2. update `nodeSelectorTerms` to match your nodes (if unsure, `kubectl describe nodes [one worker node] | grep machine-` should give you the right id for `machine-deployment` or `machine-pool` depending on your provider). Beware, there's 2 places to update!
+
+3. update `gateway.ingress.hosts.host` and `gateway.ingress.tls.host` 
+
+
 ### Deploying on AWS
 
 The recommended deployment mode is using S3 storage mode. Assuming your cluster
 has `kiam`, `cert-manager` and `external-dns` included, you should be good to use
 the instructions below to setup S3 bucket and the necessary permissions in your
 AWS account.
 
+Make sure you create this config for the *workload cluster* where you are deploying Loki.
```

Comment:
I would try to be cluster type agnostic as possible, in the end loki will be installable on any cluster (both management and workload cluster)

```suggestion
Make sure to create this config in order to deploy Loki.
```

Reply:
I agree, but I got lost between the MC and the WC's cloud-provider view, hence the specific + bold wording.
Let me try to rephrase it.